### PR TITLE
fixed LATEST with '+' bug, and also ts.del with '+'

### PR DESCRIPTION
**Pull request by OfirMos**
Fixes the following bug:
https://github.com/RedisTimeSeries/RedisTimeSeries/issues/1247

When using '+' for range or del, i.e: "TS.RANGE dailyDiffTemp:TLV - + LATEST" the determined end timestamp was wrong.

### CODE REVIEW

**Review by danni-m — DISMISSED** [review]

File: tests/flow/test_ts_delete.py [hunk 1]
```diff
@@ -146,6 +146,15 @@ def test_ts_del_multi_chunk():
             e.assertEqual(len(res), 1)
         e.flush()
 
+def test_ts_del_multi_chunk():
```
Comment (ashtul) [inline tests/flow/test_ts_delete.py:149]: Change name of test

File: tests/flow/test_ts_range.py [hunk 2]
```diff
@@ -899,6 +899,22 @@ def test_latest_flag_range():
         res = r.execute_command('TS.range', key1, 0, 20)
         assert res == [[1, '1'], [2, '3'], [11, '7'], [13, '1']] or res == [[1, b'1'], [2, b'3'], [11, b'7'], [13, b'1']]
 
+#https://github.com/RedisTimeSeries/RedisTimeSeries/issues/1247
+def test_latest_flag_range():
```
Comment (ashtul) [inline tests/flow/test_ts_range.py:903]: Change name of test

**Review by ashtul — COMMENTED** [review]
Nice work.
Small comment.

File: tests/flow/test_ts_range.py (hunk 2 above)
Comment (OfirMos) [reply tests/flow/test_ts_range.py:903]: @ashtul To which name?

File: tests/flow/test_ts_delete.py (hunk 1 above)
Comment (OfirMos) [reply tests/flow/test_ts_delete.py:149]: @ashtul To which name?

**Review by ashtul — APPROVED** [review]
:+1:

**Pull request merged**